[MI-3708]: Added the test cases for attachment sync for MS Teams sync plugin. (#12)
Brightscout/mattermost-test-management: data/test-cases/plugins/ms-teams-sync/attachments-sync/Allowed_attachment_size.md

---
# (Required) Ensure all values are filled up

> name: "Allowed size of attachment"

Reviewer: ```suggestion
name: "Allowed size of attachment on MM"
```

status: Active
priority: Normal
folder: attachments-sync

> authors: "@AayushChaudhary0001"

Reviewer: please update the author everywhere

Author: updating

team_ownership: []
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

> 1. Click on the attachments button in channel or DM/GM in MM.

Reviewer: ```suggestion
1. Upload and send an attachment of the allowed size in the linked channel or DM/GM in MM.
```

Reviewer: maybe?

Author: Yeah, we can club the selecting steps for the attachment as "uploading the attachment" and for sending the attachment i think we can define it in separate step.

2. Select the attachment of the allowed size and click on open.

> 3. Click on the send button in MM.

Reviewer: if updating, we can remove these both points

Author: clubbing the steps for selecting the attachment, as mentioned above.

4. Navigate to the linked channel or DM/GM in MS teams.

**Step 2**

1. Click on the attachments button in channel or DM/GM in MS teams.
2. Select the attachment of the allowed size and click on open.

> 3. Click on the send button in MS teams.

Reviewer: same

4. Navigate to the linked channel or DM/GM in MM.

**Expected**

> The user should be able to view the attachment in MS teams. After step 2, the user should be able to view the attachment in MM.

Reviewer: ```suggestion
The user should be able to view the attachment in MS Teams. After step 2, the user should be able to view the attachment in MM.
```

Reviewer: check everywhere

**Step 1**

> 1. Upload the attachment of the allowed size in the linked channel or DM/GM on MM.

Reviewer: can't we combine this and the below step as well?

2. Click on the send button in MM.
3. Navigate to the linked channel or DM/GM in MS Teams.

**Step 2**

> 1. Upload the attachments of the allowed size in the linked channel or DM/GM on MS Teams.

Reviewer: same. check everywhere
